scripts/rspec/rspec.ps1

```diff
@@ -56,9 +56,9 @@ if (-Not (Test-Path $resgenPath)) {
 $sonaranalyzerPath = "${PSScriptRoot}\\..\\..\\sonaranalyzer-dotnet"
 
 $categoriesMap = @{
-    "BUG" = "Sonar Bug";
-    "CODE_SMELL" = "Sonar Code Smell";
-    "VULNERABILITY" = "Sonar Vulnerability";
+    "BUG" = "Bug";
+    "CODE_SMELL" = "Code Smell";
+    "VULNERABILITY" = "Vulnerability";
 }
 
 $severitiesMap = @{
```

Comment:
What does this map do?

Reply:
Converts the SonarQube categories to displayable strings.